assets/airports/kcvg.json

```diff
@@ -0,0 +1,256 @@
+		"HNN": ["N38d45.25", "W82d01.57"],
+		"OBK": ["N42d13.28", "W87d57.10"],
+		"BDF": ["N41d09.58", "W89d35.27"],
+		"VACUB": ["N39d05.62", "W84d41.02"],
+		"NICID": ["N39d04.69", "W84d40.11"],
+		"DRUSS": ["N39d00.03", "W84d38.82"],
+		"CHCLL": ["N38d28.83", "W84d58.71"],
+		"PONIE": ["N38d19.83", "W85d03.28"],
+		"SILKS": ["N38d28.00", "W84d45.10"],
+		"KENLN": ["N38d27.45", "W84d30.47"],
+		"TRFWA": ["N38d01.03", "W84d47.43"],
+		"SKINN": ["N37d49.50", "W86d04.62"],
+		"GIPLE": ["N38d52.68", "W84d02.83"],
+		"HOBNO": ["N38d48.74", "W83d44.52"],
+		"RHOMM": ["N38d45.88", "W83d31.35"],
+		"MISSN": ["N38d45.79", "W83d00.05"],
+		"JODUB": ["N38d45.75", "W82d53.46"],
+		"LOVEY": ["N39d12.81", "W85d24.52"],
+		"ZABAL": ["N39d32.12", "W85d55.09"],
+		"MACES": ["N40d00.22", "W86d43.38"],
+		"CULED": ["N38d52.92", "W84d52.30"],
+		"FANSA": ["N38d57.25", "W84d42.76"],
+		"FEBLU": ["N38d53.86", "W84d41.08"],
+		"FLOEE": ["N38d57.40", "W84d40.15"],
+		"ZEGUL": ["N38d49.74", "W84d35.12"],
+		"CAMUM": ["N38d57.45", "W84d37.89"],
+		"FRTZY": ["N39d06.12", "W84d55.20"],
+		"CEMLA": ["N39d13.32", "W84d48.99"],
+		"EVAME": ["N39d13.38", "W84d43.66"],
+		"HAGSS": ["N39d09.40", "W84d38.80"],
+		"BOUMR": ["N39d06.29", "W84d52.23"],
+		"HOBAL": ["N39d10.13", "W84d45.43"],
+		"BONLE": ["N39d02.77", "W84d50.31"],
+		"ANTEO": ["N38d56.09", "W84d34.61"],
+		"CODKO": ["N38d56.09", "W84d30.00"],
+		"JBETH": ["N38d59.90", "W84d26.29"],
+		"FOPAS": ["N39d05.28", "W84d24.89"],
+		"ZUGLU": ["N38d55.55", "W84d45.20"],
+		"MIPIE": ["N38d55.99", "W84d48.83"],
+		"HERMM": ["N38d58.69", "W84d52.82"],
+		"AMMOT": ["N39d11.92", "W84d52.33"],
+		"CIGMA": ["N39d07.90", "W84d43.14"],
+		"DJOHN": ["N39d11.54", "W84d43.10"],
+		"JBNCH": ["N39d34.88", "W85d01.35"],
+		"HAGOL": ["N39d34.15", "W84d35.60"],
+		"BNGLE": ["N39d33.01", "W84d24.52"],
+		"RIKLE": ["N39d49.84", "W84d07.30"]
+	},
+	"runways": [{
+		"name": ["09", "27"],
+		"name_offset": [
+			[0, 0],
+			[0, 0]
+		],
+		"end": [
+			["N39d2.78", "W84d41.71", "883.3ft"],
+			["N39d2.78", "W84d39.17", "874.9ft"]
+		],
+		"delay": [10, 10],
+		"ils": [true, true]
+	}, {
+		"name": ["18C", "36C"],
+		"name_offset": [
+			[0, 0],
+			[0, 0]
+		],
+		"end": [
+			["N39d3.88", "W84d40.12", "874.6ft"],
+			["N39d2.07", "W84d40.12", "840.9ft"]
+		],
+		"delay": [10, 10],
+		"ils": [true, true]
+	}, {
+		"name": ["18L", "36R"],
+		"name_offset": [
+			[0, 0],
+			[0, 0]
+		],
+		"end": [
+			["N39d3.35", "W84d38.80", "886.2ft"],
+			["N39d1.70", "W84d38.81", "896.2ft"]
+		],
+		"delay": [10, 10],
+		"ils": [true, true]
+	}, {
+		"name": ["18R", "36L"],
+		"name_offset": [
+			[0, 0],
+			[0, 0]
+		],
+		"end": [
+			["N39d4.25", "W84d41.02", "865.4ft"],
+			["N39d2.94", "W84d41.03", "873.4ft"]
+		],
+		"delay": [10, 10],
+		"ils": [true, true]
+	}],
+	"sids": {
+		"CHCLL4": {
+			"icao": "CHCLL4",
+			"name": "Churchill Four",
+			"rwy": {
+				"18L": ["CAMUM", "ZEGUL"],
+				"18R": ["FANSA"],
+				"18C": [
```

Comment:
with routes, we _break_ some syntax rules, this is an instance of that.  we like to keep this all on one line  (when possible):
```javascript
"18C": [["FLOEE", "A14+"], "FEBLU"],
```